src/zenml/zen_server/auth.py

```diff
@@ -609,9 +609,12 @@ def authenticate_external_user(external_access_token: str) -> AuthContext:
         with AnalyticsContext() as context:
             context.user_id = user.id
             context.identify(
-                traits={"email": user.email, "source": "external_auth"}
+                traits={
+                    "email": external_user.email,
+                    "source": "external_auth",
+                }
             )
-            context.alias(user_id=user.id, previous_id=external_user.id)
+            context.alias(user_id=external_user.id, previous_id=user.id)
```

Comment:
If I recall correctly, changing the order of these two messed up the identification process before when we first implemented the first version. Why do we have to change this now?

Reply:
I think this is actually the wrong way round and messed with ortto as far as I can see.